backend/pokebattle/settings/base.py

```diff
@@ -120,3 +120,7 @@ def base_dir_join(*args):
 # Sentry
 SENTRY_DSN = config("SENTRY_DSN", default="")
 COMMIT_SHA = config("HEROKU_SLUG_COMMIT", default="")
+
+# Email
+EMAIL_ADDRESS = "jaenia@vinta.com.br"
```

Comment:
Ideally you'd set this in a settings variable. We don't want to hard code this kind of information